stdlib/3/_thread.pyi

```diff
@@ -1,15 +1,24 @@
 # Stubs for _thread
 
-# NOTE: These are incomplete!
+from typing import Any, Callable, Dict, NoReturn, Tuple
 
-from typing import Any
+error = RuntimeError
 
 def _count() -> int: ...
 _dangling = ...  # type: Any
 
 class LockType:
-    def acquire(self) -> None: ...
+    def acquire(self, blocking: bool = ..., timeout: float = ...) -> bool: ...
     def release(self) -> None: ...
+    def locked(self) -> bool: ...
+    def __enter__(self) -> bool: ...
+    def __exit__(self) -> None: ...
```

Comment:
Thank you, one more thing:

```suggestion
    def __exit__(self, type: Optional[Type[BaseException]], value: Optional[BaseException], traceback: Optional[types.TracebackType]) -> None: ...
```

(with `import types`)

Reply:
(It seems the implementation in `_threadmodule.c` actually allows any number of arguments with any type and just disregards them. On the other hand, `_dummy_thread.py` required any three arguments, so it may make some sense to use either of that.)